tgui/packages/tgui/interfaces/TurbineComputer.js

```diff
@@ -0,0 +1,57 @@
+import { useBackend } from '../backend';
+import { Button, LabeledList, Section} from '../components';
+import { Window } from '../layouts';
+
+export const TurbineComputer = (props, context) => {
+  const { act, data } = useBackend(context);
+  const operational = Boolean(data.compressor && !data.compressor_broken && data.turbine && !data.turbine_broken);
```

Comment:
instead of accessing data like `data.VAR_NAME` you should instead cast them to consts like this
```js
const {
  compressors,
  compressors_broken,
  turbine,
  etc
} = data;
```
so that you can just use the data directly. i.g. `VAR_NAME` instead of `data.VAR_NAME`

Reply:
Would I be getting rid of the use of operational in this case?

Reply:
no, operational is just fine, just make sure that var definition goes after the data const. It's actually a perfect way to handle what you're trying to do IMO

Reply:
Resolved with commit [0d2f7cf](https://github.com/ParadiseSS13/Paradise/pull/21226/commits/0d2f7cfb794235001e46beb41a04e1cba50fbdc1)